### PR TITLE
[reward] Increase percision on percentage in schedule

### PR DESCRIPTION
**Pull request by fxfactorial**
I think this means we need to restart the testing networks from scratch

### CODE REVIEW

File: consensus/reward/schedule.go [hunk 1]
```diff
@@ -18,86 +18,87 @@ var (
...
-		mustParse("2019-May-31"): numeric.MustNewDecFromStr("0.2429"),
-		mustParse("2019-Jun-30"): numeric.MustNewDecFromStr("0.2488"),
-		mustParse("2019-Jul-31"): numeric.MustNewDecFromStr("0.2547"),
-		mustParse("2019-Aug-31"): numeric.MustNewDecFromStr("0.2607"),
-		mustParse("2019-Sep-30"): numeric.MustNewDecFromStr("0.2666"),
-		mustParse("2019-Oct-31"): numeric.MustNewDecFromStr("0.2726"),
-		mustParse("2019-Nov-30"): numeric.MustNewDecFromStr("0.3785"),
-		mustParse("2019-Dec-31"): numeric.MustNewDecFromStr("0.3844"),
+		mustParse("2019-May-31"): numeric.MustNewDecFromStr("0.242864761904762"),
+		mustParse("2019-Jun-30"): numeric.MustNewDecFromStr("0.254747114845938"),
```
Comment (rlan35) [inline consensus/reward/schedule.go]: This line is wrong. Did you programmatically did the change?

Comment (rlan35) [reply consensus/reward/schedule.go]: Please double check. The number is critical.

Comment (fxfactorial) [reply consensus/reward/schedule.go]: yes, programmaticaly. Multiple line copy-paste in emacs. Just I did the multiple line copy paste after that line, (initially was doing by hand on the first 2)

Comment (rlan35) [reply consensus/reward/schedule.go]: ok

**Comment by AndyBoWu** [comment]
@fxfactorial "I think this means we need to restart the testing networks from scratch", do we need to do that? please confirm

**Pull request merged**